### ISSUE TITLE
PersonGender Crash with GetTvSeasonAsync

### ISSUE THREAD

**Issue by FrederikBolding** [LordMike/TMDbLib#220]
I seem to be getting this crash recently when using GetTvSeasonAsync: 
`Error converting value {null} to type 'TMDbLib.Objects.People.PersonGender'. Path 'episodes[0].crew[0].gender', line 1, position 181.`

It might have been due to a recent update, and it also seems like it only affects newer shows.

It can be fixed by specifying a custom serializer 
`new JsonSerializer { NullValueHandling = NullValueHandling.Ignore}`

But is crashing with the default configuration.

**Comment by Naliath** [comment]
Yea noticed some gender related changes in a previous issue. I'll take a peak tomorrow see if I can fix it in our processing.

**Comment by Naliath** [comment]
Quick peek on my phone, figure best way is to simply make it nullable. Since well clearly it is on the api side of things :)

**Comment by Naliath** [comment]
Sweating my ass off due to a nice heatwave we have going on, hence the delay. But I just added a new enum converter that the lovely Stackoverflow gods provided. With a few minor changes to get it to play nice with .NET Standard 1.2 it should now be ready to rock.

The tests seem to be running fine again so shortly a new version should be popping up on nuget.

I opted for this approach since we provided the unknown value already and as such this change does not change our contract. Additionally this keeps it nice and easy to deal with null values.

@LordMike Seems like there where more changes regarding lists, account states and people search. We will need to take a look at those sections as well to get then back in line
